clientSrc/components/Map.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-alert */
+import React from 'react';
+import '../styles/Map.less';
+
+class Map extends React.Component {
```

Comment:
prefered way is to use function hooks instead of class components

Reply:
I'm still way in the beginner stage and am still learning hooks. Once we're a little further, I'm going to convert the class components to hooks in a batch.